google-basemap/README.md

```diff
@@ -0,0 +1,18 @@
+## Example: Google maps deck.gl and CARTO
+Uses [Vite](https://vitejs.dev/) to bundle and serve files.
+
+## Usage
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/CartoDB/deck.gl-examples/tree/master/basemaps?file=index.ts)
```

Comment:
Wrong link, rename the folder of the example to `google-maps`

Reply:
Done